rider-fsharp/testData/templates/net60/Net60/XUnitProjectTemplateTest/projectModel/gold/projectModel_win.gold

```diff
@@ -29,7 +29,7 @@ Project [Name:Miscellaneous Files, GUID:GUID]
 
 Project [Name:UnitTestProject, GUID:GUID]
   Child items:
-    ProjectFileImpl(Path : Microsoft.TestPlatform.PlatformAbstractions.dll (../../../../../../../.nuget/packages/microsoft.testplatform.testhost/17.1.0/build/netcoreapp2.1/x64/Microsoft.TestPlatform.PlatformAbstractions.dll))[Hidden CONTENT ]
+    ProjectFileImpl(Path : Microsoft.TestPlatform.PlatformAbstractions.dll (../../../113942792/dotnetHomeCli/.nuget/packages/microsoft.testplatform.testhost/17.1.0/build/netcoreapp2.1/x64/Microsoft.TestPlatform.PlatformAbstractions.dll))[Hidden CONTENT ]
```

Comment:
Is it safe to commit the `113942792` string? It looks like something that can change and/or be different on different machines.